scripts/coordinate.js

```diff
@@ -363,6 +363,12 @@ FriendlyChat.prototype.getSession = function() {
         let renderWhenInformation = data.val().whenDescription;
         let renderWhereInformation = data.val().whereDescription;
         let renderWhoInformation = data.val().whoDescription; // To-Do: Update with first names dynamically.
+        let renderWhoAmIInformation = data.val().whoAmIDescription;
+        //for DoWhops created before Who Am I was added to create field
+        //do an undefined check so it doesn't render an undefined paragraph
+        if (renderWhoAmIInformation === undefined) {
```

Comment:
This may be a consequence of us having older data in the db that doesn't have the parameter. Yeah, let's check it out tomorrow @mike-mjk